CHANGELOG.md

```diff
@@ -1,11 +1,35 @@
-# 0.3.0 - 2021/03/26
+## 1.2.0
+
+* Add support for a TALKTOME_LAYOUTS_FOLDER environment variable used
+  in auto_options. When set, the folder is used to embed formatted emails
+  in more general layouts (see Client::Local.templater).
```

Comment:
I'm afraid there may be confusion between layout and templates for people like me.